cms/datavis/blocks/base.py

```diff
@@ -146,12 +144,11 @@ def get_x_axis_config(
         # x-axis title default value is undefined. See
         # https://api.highcharts.com/highcharts/xAxis.title.text
         if (title := attrs.get("title")) and getattr(self, "supports_x_axis_title", False):
-            config["title"] = {
-                "text": title,
-            }
+            config["title"] = title
 
         if (tick_interval := attrs.get("tick_interval")) is not None:
-            config["tickInterval"] = tick_interval
+            config["tickIntervalMobile"] = tick_interval
```

Comment:
These changes don't seem relevant to the PR

Reply:
DS version 72.8.0 introduces configuralble tick intervals as separate mobile and desktop parameters in https://github.com/ONSdigital/design-system/pull/3551. This PR is about compatibility with a new version of the Design System. If not here, then when and how should we update these parameters in the chart builder?

Reply:
Oh I see - I thought they had crept in from somewhere else. Ignore me in that case.

Reply:
Understood, thanks.